data/formats-data.js

```diff
@@ -5640,7 +5647,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	swirlix: {
-		randomBattleMoves: ["calmmind", "drainingkiss", "dazzlinggleam", "surf", "psychic", "flamethrower", "bellydrum", "thunderbolt", "return", "thief", "cottonguard"],
+		randomBattleMoves: ["flamethrower", "bellydrum", "playrough", "return", "thief", "defog", "stickyweb"],
```

Comment:
draining kiss, calmmind, psychic for special set

Reply:
as w/ vullaby im worried for option overload

Reply:
hm that's true - xerneas is a good example of bad movesets all the time